Sources/MixpanelInstance.swift

```diff
@@ -795,7 +793,7 @@ extension MixpanelInstance {
                 } else {
                     self.people.distinctId = nil
                 }
-            }
+            
```

Comment:
minor nit: pls format the code. otherwise, it looks good.

Reply:
@zihejia I've fixed the formatting :)

Reply:
I've also added a fix to also encode "+" within URLs which was causing some decide requests not to fetch data.